backend/tracim_backend/lib/agenda/agenda.py

```diff
@@ -193,7 +193,7 @@ def get_user_agendas(
             workspace_api = WorkspaceApi(
                 current_user=self._user, session=self._session, config=self._config
             )
-            workspaces = workspace_api.get_all_for_user(user)
+            workspaces = workspace_api.get_all_for_user(user, with_role=True, owned=False)
```

Comment:
Why exclude owned workspaces here ?

Reply:
This code excludes only workspace where user is owner and has no role.
We do not expect user without any role to access to agenda feature. This change is just to obtain same behavior as before.

Reply:
Okay I got it. `with_role` and `owned` permit to include them. If both are set to False, list is always empty.
Can you rename into `include_with_role` and `include_owned` and add a docstring to `get_all_for_user` please ?

Reply:
done.